index.html

```diff
@@ -9,13 +10,82 @@
     <link rel="stylesheet" href="css/normalize.css">
     <!-- Add a link to your CSS file here (use the line above to guide you) -->
     <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
 </head>
+
 <body>
+    <header>
+        <img src="img/karma-logo.svg" class="logo">
+        <nav>
+            <ul class="list">
+                <li><b>Meet Karma</b></li>
+                <li>How it Works</li>
+                <li>Store</li>
+                <li>Blog</li>
+                <li>Help</li>
+                <li>Login</li>
+            </ul>
+        </nav>
+        <img src="img/menu-hamburger.svg" alt="Hamburger Menu" class="hamburger">
+    </header>
+
+
+    <main>
+        <section class="backgroundImg">
+
+            <h1 class="firstText">Introducing Karma</h1>
+            <p class="secondText">Bring WiFi with you, everywhere you go.</p>
+            <button class="buttonS" type="submit">Learn More</button>
```

Comment:
Watch out for the casing (buttonS should have a small s)

Reply:
It's good to have added a type to your button, however in this case the button isn't submitting a form, it's more like a link to another page. So you don't need it, or you could use `type="button"`